vignettes/precompile.R

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+
+
+
+# Precompile vignettes
+
+
+library(knitr)
+
+
+# Convert *.orig to *.Rmd -------------------------------------------------
+
```

Comment:
This is all good, but if get more than two vignettes in the future we can loop this and use `basename` etc.

Reply:
I went ahead and generalized this. Good idea.

Reply:
Beautiful!